### PR TITLE
Added recipe for fa2 package

### PR DESCRIPTION
**Pull request by msadang**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

File: recipes/fa2/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+    - fa2
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://github.com/bhargavchippada/forceatlas2
+  summary: 'The fastest ForceAtlas2 algorithm for Python (and NetworkX)'
+  license: GPL-2.0-or-later
```
Comment (jakirkham) [inline recipes/fa2/meta.yaml:41]: Appears to be GPL 3

```suggestion
  license: GPL-3.0-or-later
```

File: recipes/fa2/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+  license_file: LICENSE
```
Comment (jakirkham) [inline recipes/fa2/meta.yaml:42]: Asked upstream to package the license file in the future so we can just use it directly

xref: https://github.com/bhargavchippada/forceatlas2/issues/30

File: recipes/fa2/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,48 @@
+{% set name = "fa2" %}
+{% set version = "0.3.5" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/fa2-{{ version }}.tar.gz
```
Comment (jakirkham) [inline recipes/fa2/meta.yaml:10]: Taken from [the example]( https://github.com/conda-forge/staged-recipes/blob/2efe131304e7f0ca00bad5168cabf657086da12b/recipes/example/meta.yaml#L14 ).

```suggestion
  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
```

**Review by jakirkham — COMMENTED** [review]
Thanks @msadang! Also welcome to conda-forge 😄

Looks pretty good. Made a few suggestions below. Please let us know if you have any questions 🙂

File: recipes/fa2/meta.yaml (hunk 3 above)
Comment (msadang) [reply recipes/fa2/meta.yaml:10]: Change completed.

File: recipes/fa2/meta.yaml (hunk 1 above)
Comment (msadang) [reply recipes/fa2/meta.yaml:41]: Change completed.

**Comment by jakirkham** [comment]
@conda-forge/staged-recipes, this is ready for review! 😄

**Pull request merged**